dask_bigquery/tests/test_core.py

```diff
@@ -66,12 +71,96 @@ def dataset(df):
         )
 
 
+@pytest.fixture
+def write_dataset():
+    credentials, project_id = google.auth.default()
+    env_project_id = os.environ.get("DASK_BIGQUERY_PROJECT_ID")
+    if env_project_id:
+        project_id = env_project_id
+
+    dataset_id = uuid.uuid4().hex
+
+    yield credentials, project_id, dataset_id
+
+    with bigquery.Client() as bq_client:
+        bq_client.delete_dataset(
+            dataset=f"{project_id}.{dataset_id}",
+            delete_contents=True,
+        )
+
+
+@pytest.mark.parametrize(
+    "parquet_kwargs,load_job_kwargs",
+    [
+        (None, None),
+        (
+            {
+                "schema": pa.schema(
+                    [
+                        ("name", pa.string()),
+                        ("number", pa.uint8()),
+                        ("timestamp", pa.timestamp("ns")),
+                        ("idx", pa.uint8()),
+                    ]
+                ),
+                "write_index": False,
```

Comment:
Does this make sense? This is set every time anyway?

Reply:
No, this one can be overridden by the user.

Reply:
Good point, but False is the default, isn't it?